src/cnd/cnd.ts

```diff
@@ -44,6 +44,56 @@ export interface SwapRequest {
   peer: Peer;
 }
 
+export interface CoreRequestBody {
+  alpha_amount: string;
+  beta_amount: string;
+  alpha_network: string;
+  beta_network: string;
+  alpha_identity: string;
+  beta_identity: string;
+  role: "Alice" | "Bob";
+  peer: Peer;
+}
+
+export interface AlphaHanAdditionalRequestBody {
+  alpha_absolute_expiry: number;
+}
+
+export interface BetaHanAdditionalRequestBody {
+  beta_absolute_expiry: number;
+}
+
+export interface AlphaHerc20AdditionalRequestBody {
+  alpha_contract_address: string;
```

Comment:
Make this more obvious?

```suggestion
  alpha_erc20_contract_address: string;
```

Reply:
Yes I hesitated. Went for the simpler version for 2 reasons:
- Simpler is better
- The information that it is erc20 is already present in the URL